sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -743,12 +743,12 @@ private void downloadToFileCleanup(AsynchronousFileChannel channel) {
      * @param accessConditions {@link BlobAccessConditions}
      * @return A reactive response signalling completion.
      */
-    public Mono<Response<Void>> setHTTPHeadersWithResponse(BlobHTTPHeaders headers,
+    public Mono<Response<Void>> setHttpHeadersWithResponse(BlobHTTPHeaders headers,
```

Comment:
Is the BlobHTTPHeaders class going to change too?

Reply:
This is a codegen class, not sure how possible it is to fix this casing

Reply:
@jianghaolu Is this something that can be fixed in autorest?